drivers/mulesoft/calendar_exporter.cr

```diff
@@ -34,12 +34,13 @@ class MuleSoft::CalendarExporter < PlaceOS::Driver
 
     @time_zone_string = setting?(String, :calendar_time_zone).presence
     @time_zone = Time::Location.load(@time_zone_string.not_nil!) if @time_zone_string
-    self[:timezone] = Time.local.to_s
+    self[:timezone] = @time_zone.to_s
 
     subscription = system.subscribe(:Bookings_1, :bookings) do |_subscription, mulesoft_bookings|
-      logger.debug { "DETECTED changed in Mulesoft Bookings.." }
+      logger.debug { "DETECTED change in Mulesoft Bookings..." }
       @bookings = Array(Hash(String, Int64 | String | Nil)).from_json(mulesoft_bookings)
```

Comment:
Doing something along these lines instead of the `Hash` would be much easier to maintain, faster, and ensures correctness

```crystal
record MulesoftBooking,
  getter title : String?,
  getter recurring_master_id : Int64,
  getter event_start : Int64,
  getter event_end : Int64,
  getter body : String do

  include JSON::Serializable

  def to_place_calendar(
    user_id : String,
    attendees : Array(String),
    timezone : String,
    system_email : String,
    system_name : String,
  )
    {
      title:       "#{recurring_master_id} #{title}",
      event_start: event_start,
      event_end:   event_end,
      timezone:    timezone,
      description: body,
      user_id:     system_email,
      location:    system_name,
      attendees:   [{email: system_email, name: system_name}],
    }
  end
end
```

Reply:
Thanks so much, I'll grok this after some other tasks, then see if anything else should change and then test.
Yeah I was originally intending the booking hash and calendar event to be the same object but couldn't get that to work just due to my crystal inexperience. Ran out of time due to a delivery deadline so just implemented it this way.
Now that I have time it'll be great (and very educational for me) to implement it properly.